requirements.txt

```diff
@@ -16,7 +16,7 @@ Click==7.0
 click-didyoumean==0.3.0
 click-plugins==1.1.1
 click-repl==0.2.0
-codecov==2.1.12
```

Comment:
looks like codecove 2.1.12 is deprecated. They replaced it with 2.1.13 according to https://about.codecov.io/blog/message-regarding-the-pypi-package/